packages/strace/project.bri

```diff
@@ -14,7 +14,18 @@ const source = Brioche.download(
 
 export default function strace(): std.Recipe<std.Directory> {
   return std.runBash`
-    ./configure --prefix=/
+    case "$(uname -m)" in
```

Comment:
We can instead use `--enable-mpers=check`, see https://github.com/strace/strace/commit/3eb199a29d29c2cb44b66483d305717006b72b7f. This option let's detect if the current architecture does support multiple personalities.

Reply:
Might be a toolchain issue or maybe it's simply a limitation on this architecture. I wasn't't able to find any documentation on it. Except that in Nixpkgs, the flag is always set: https://github.com/NixOS/nixpkgs/blob/c7ab75210cb8cb16ddd8f290755d9558edde7ee1/pkgs/by-name/st/strace/package.nix#L42 and on Homebrew, they simply disabled it (but that's not the right option: https://github.com/Homebrew/homebrew-core/blob/29857a21b7560c2e0bb826b183cfaa8b0c1871fe/Formula/s/strace.rb#L24

Sure, we can leave it as is, and see later by opening an issue.